query-engine/query-engine-wasm/Cargo.toml

```diff
@@ -8,6 +8,11 @@ doc = false
 crate-type = ["cdylib"]
 name = "query_engine_wasm"
 
+[features]
+sqlite = ["driver-adapters/sqlite", "sql-connector/sqlite"]
+postgresql = ["driver-adapters/postgresql", "sql-connector/postgresql"]
```

Comment:
Here and in all other crates, feature is named `postgresql` rather than `postgres` for consistency with `quaint`: there, already existing feature is also called `postgresql`. Not the biggest fan of the name myself, couple of alternative suggestions:
- Rename `quaint` feature to `postgres`
- Continue using `postgresql` in `quaint`, but `postgres` everywhere else.

Reply:
See commment in https://github.com/prisma/prisma-engines/pull/4681/files#diff-17480bbc56bcae20b2ceaf4976b39cf5b6f608d1222448aa48fb2e660d0b901aR25